### ISSUE TITLE
overfitting issues

### ISSUE THREAD

**Issue by ak9250** [mit-han-lab/data-efficient-gans#19]
"The training length (default to 300k images) may be increased for larger datasets; note that there may be overfitting issues if the training is too long."

for too long training, is there any metric for this based on the dataset size?

**Comment by zsyzzsoft** [comment]
This is observed usually when FID stops decreasing.

**Comment by ak9250** [comment]
@zsyzzsoft thanks, so for finetuning for example on n=30, once fid score starts going up would stop training?
<img width="847" alt="Screen Shot 2020-09-29 at 12 58 52 AM" src="https://user-images.githubusercontent.com/6818285/94563224-63bfc800-0234-11eb-8c8f-feed06e59c4c.png">

**Comment by zsyzzsoft** [comment]
I would suggest stopping training if FID keeps increasing or does not go below the minimum over 3 or 5 checkpoints.
